recipes-go/3mdeb-rtectrl/3mdeb-rtectrl/rte_ctrl.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
```

Comment:
This script is already here: https://github.com/3mdeb/RteCtrl/pull/7/files
Why duplicate it?

Reply:
I see it is removed so will resolve conversation